docs/changelog.txt

```diff
@@ -50,7 +50,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 
 ## API
 - `buildingplan`: added Lua interface API
-- `job-module`: add item type param to dfhack.job.isSuitableMaterial() so the non_economic flag can be properly handled (it was being matched for all item types instead of just boulders)
+- add item type param to dfhack.job.isSuitableMaterial() so the non_economic flag can be properly handled (it was being matched for all item types instead of just boulders)
```

Comment:
We could add an anchor like `.. _job-module:` to `Lua API.rst` (although it would only document what's exposed to Lua). The reason I don't usually do this is that changelogs are essentially reused verbatim as Markdown for GitHub and Reddit, causing anything in backticks to show up as code. It's fine for tools like `buildingplan` but could be weird or confusing for `job-module` to show up as-is. (I manually corrected the quickfort user guide link for r3, but a longer-term solution would be to actually generate proper markdown links somehow.)

An alternative could be to start with ``dfhack.job.isSuitableMaterial()`` (in double backticks, so it's not an RST link). Doesn't need to be part of this PR - I plan to work on changelog-related documentation before r4, so I'll do it then.

Reply:
ok, I'll use the double backtick approach in the future in situations like this. Thanks!